modules/storage/local.go

```diff
@@ -107,8 +119,19 @@ func (l *LocalStorage) Stat(path string) (os.FileInfo, error) {
 	return os.Stat(filepath.Join(l.dir, path))
 }
 
+func isLocalPathValid(path string) bool {
+	a := filepath.Clean(path)
+	if strings.HasPrefix(a, fmt.Sprintf("..%c", filepath.Separator)) {
```

Comment:
Reading the fourth point from `filepath.Clean` this condition should always be false.

Reply:
`filepath.Clean("../todo.txt") == "../todo.txt"`

Reply:
Dammit, reading golang's std library documentation gets harder by the day.